prov/verbs/src/verbs_rma.c

```diff
@@ -508,7 +508,7 @@ static ssize_t fi_ibv_rdm_ep_rma_inject_write(struct fid_ep *ep,
 	struct fi_ibv_rdm_rma_start_data start_data = {
 		.conn = conn,
 		.ep_rdm = ep_rdm,
-		.data_len = (uint32_t)len,
+		.data_len = (uint64_t)len,
```

Comment:
In this case (inject) just to be aligned with fi_ibv_rdm_rma_start_data.data_len, msx_inject_size much less INT_MAX
But in common case, RDM/RMA currently supports more then 1GB transfers by segmentation.